packages/requester-utils/src/RequesterUtils.ts

```diff
@@ -149,12 +171,13 @@ export function createRequesterFn(
 
   return (serviceOptions) => {
     const requester: RequesterType = {} as RequesterType;
+    const rateLimiters = createRateLimiters(serviceOptions.rateLimits);
 
     methods.forEach((m) => {
       requester[m] = async (endpoint: string, options: Record<string, unknown>) => {
         const requestOptions = await optionsHandler(serviceOptions, { ...options, method: m });
 
-        return requestHandler(endpoint, requestOptions);
+        return requestHandler(endpoint, { ...requestOptions, rateLimiters });
```

Comment:
I think that you might want to make this something like: `requestHandler(endpoint, requestOptions, opts)` where `opts` is a named set of additional parameters that can be expanded over time.

```
opts = { rateLimiters: <whatever>, myNewOpt: <something_else> }
```

The first two parameters are set and you run the risk of something from `rateLimiters` overwriting a setting from `requestOptions` in the current implementation.

It's nice when all of the parameters are named (or positional) but that would be a breaking change and probably not worth it at this point.

Reply:
Thought about that but i figured i would just expand the "requestOptions" variable, which is what ive been doing thus far. Normally id include this logic in that optionsHandler function but I didnt want it to reinitialize everytime a request was made.

Reply:
Yeah, that's fair. It's all down in the weeds so a later refactor shouldn't mess anyone up if you decide to do it.